MAUI/DataGrid/filtering.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: post
+title: Filtering in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about Filtering support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control and more.
+platform: MAUI
+control: SfDataGrid
+documentation: UG
+---
+
+# Filtering in MAUI DataGrid (SfDataGrid)
+
+Filtering is the process of retrieving the values from the collection which satisfy the specified condition. The [SfDataGrid](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html) provides the view filtering support.
+
+## View filtering
+
+The `SfDataGrid` supports filtering the records in the view by setting [SfDataGrid.View.Filter](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.ICollectionViewAdv.html#Syncfusion_Maui_Data_ICollectionViewAdv_Filter) property where `Filter` is a predicate.
+
+>N Note: In order to refresh filtering for the newly added row or column, set the [SfDataGrid.View.LiveDataUpdateMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.ICollectionViewAdv.html#Syncfusion_Maui_Data_ICollectionViewAdv_LiveDataUpdateMode) to [LiveDataUpdateMode.AllowDataShaping](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.LiveDataUpdateMode.html#Syncfusion_Maui_Data_LiveDataUpdateMode_AllowDataShaping).
+
+{% tabs %}
+{% highlight c# %}
+private void Button_Clicked(object sender, EventArgs e)
+{
+	this.sfDataGrid.View.Filter = FilterRecords;
+    this.sfDataGrid.View.RefreshFilter();
+}
+
+public bool FilterRecords(object record)
+{
+	OrderInfo? orderInfo = record as OrderInfo;
+
+	if(orderInfo != null && orderInfo.ShipCity == "Germany")
+	{
+		return true;
+	}
+	return false;
+}
+{% endhighlight %}
+{% endtabs %}
+
+>N Note: View filtering is not supported when [ItemsSource](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_ItemsSource) is [DataTable](https://docs.microsoft.com/en-us/dotnet/api/system.data.datatable?view=net-6.0).

```

Comment:
>N Note: Viewing filtering is not supported when the [ItemsSource](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_ItemsSource) is [DataTable](https://docs.microsoft.com/en-us/dotnet/api/system.data.datatable?view=net-6.0).

Reply:
It's the technical name of the feature. So, we can't change it to Viewing filtering.